src/components/organisms/Settings/ProjectList/hooks.ts

```diff
@@ -28,9 +28,7 @@ export default () => {
   const openModal = useCallback(() => setModalShown(true), []);
 
   const { data, loading, refetch } = useMeQuery();
-  const [createNewProject] = useCreateProjectMutation({
-    refetchQueries: ["Project"],
-  });
+  const [createNewProject] = useCreateProjectMutation();
```

Comment:
You are turning off refetch, did you have any problems with it?

Reply:
Beacuse it refect below inside the createProject function, and also it didn't work here.

Reply:
on line: 100 , it refetched, so I don't need refetch here.. @rot1024 
      
const scene = await createScene({
        variables: { projectId: project.data.createProject.project.id },
        refetchQueries: ["Me"],
      });